eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -84,12 +127,14 @@ stages:
                     steps:
                       - checkout: self
                       - pwsh: |
-                          $transformedArtifact = "${{artifact.name}}".Replace("azure-", "")
-                          $artifactLocation = "$(Pipeline.Workspace)/${{parameters.DocArtifact}}/$($transformedArtifact).zip"
-                          New-Item -Type Directory -Name ${{artifact.safeName}} -Path $(Pipeline.Workspace)
-                          New-Item -Type Directory -Name ${{parameters.DocArtifact}} -Path $(Pipeline.Workspace)/${{artifact.safeName}}
-                          Copy-Item  $artifactLocation $(Pipeline.Workspace)/${{artifact.safeName}}/${{parameters.DocArtifact}}
-                        displayName: Stage artifacts
+                          $adjustedName = "${{artifact.name}}".Replace("azure-", "")
+                          Write-Host "##vso[task.setvariable variable=Documentation.Zip]$adjustedName"
+                        displayName: Set Documentation File Name
+                      - template: /eng/pipelines/templates/steps/stage-artifacts.yml
+                        parameters:
+                          SourceFolder: ${{parameters.DocArtifact}}
+                          TargetFolder: ${{artifact.safeName}}/${{parameters.DocArtifact}}
+                          PackageName: $(Documentation.Zip)
                       - pwsh: |
                           Get-ChildItem -Recurse $(Pipeline.Workspace)/${{artifact.safeName}}
```

Comment:
If we feel it is interesting to keep this dir perhaps we should just add it to the template.

Reply:
Yeah. I should have done that. I'm not making more changes to this PR unless they're really necessary though.

Reply:
Not necessary for this PR but as we go back and clean-up some of these release pipelines we should consider it then.